wellcomeml/ml/bilstm.py

```diff
@@ -1,10 +1,21 @@
+from datetime import datetime
+
 from sklearn.model_selection import train_test_split
 from sklearn.base import BaseEstimator, ClassifierMixin
 from sklearn.metrics import f1_score
 import tensorflow as tf
 
 from wellcomeml.ml.attention import HierarchicalAttention
-from wellcomeml.ml.keras_utils import Metrics
+
+TENSORBOARD_LOG_DIR = "logs/scalars/" + datetime.now().strftime("%Y%m%d-%H%M%S")
+METRIC_DICT = {
+    'precision': tf.keras.metrics.Precision(name='precision'),
+    'recall': tf.keras.metrics.Recall(name='recall'),
+    'auc': tf.keras.metrics.AUC(name='auc')
+}
+CALLBACK_DICT = {
```

Comment:
@aCampello and @lizgzil, i have some second thoughts about the name of this variable. is it clearer if i call them dispatcher as @aCampello has done in other places. any names that could avoid using the word dict

Reply:
I don't have a strong opinion about it. I think dispatch is an old name that comes from other languages, so I often use it but wouldn't say it is best naming practice. It is often when the arguments are functions/callables (I might have abused the notation sometimes).

Other options: `METRICS`, `METRIC_MAP`, `NAME_TO_METRIC` or something..


Perhaps `METRIC_MAP`